include/picongpu/param/png.param

```diff
@@ -46,10 +46,18 @@ namespace picongpu
         // 5: BlowOut: typical fields, assuming that a LWFA in the blowout
         //             regime causes a bubble with radius of approx. the laser's
         //             beam waist (use for bubble fields)
+        // 6: Custom:  user-provided normalization factors via customNormalizationSI
 #define EM_FIELD_SCALE_CHANNEL1 -1
 #define EM_FIELD_SCALE_CHANNEL2 -1
 #define EM_FIELD_SCALE_CHANNEL3 -1
 
+        /** SI values to be used for Custom normalization
+         *
+         * The order of normalization values is: E, B, current (note - current, not current density).
+         * This variable must always be defined, but has no effect for other normalization types.
+         */
+        constexpr float_64 customNormalizationSI[3] = {5.0e12, 5.0e12 / SI::SPEED_OF_LIGHT_SI, 15.0};
```

Comment:
Could it be that this normalization value must be in `pic` units?

Reply:
No, this is in SI, as all our input quantities are. Internally, I convert to our units inside `typicalFields<6>::get()`. I tested the implementation, these default values are for a laser with A0 = 1.